core/services/relay/evm/types/int_types_gen.go

```diff
@@ -0,0 +1,2205 @@
+package types
+
+import (
+	"math/big"
+	"reflect"
+
+	"github.com/fxamacker/cbor/v2"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/types"
+)
+
+type SizedBigInt interface {
+	Verify() error
+	private()
```

Comment:
what is `private()` used for?

Reply:
I wanted to be sure that nothing can implement the interface that wasn't generated.